libs/blocks/card/card.js

```diff
@@ -17,20 +17,31 @@ const getCardType = (styles) => {
   return cardTypes[authoredType] || HALF;
 };
 
+const getUpFromSectionMetadata = (section) => {
+  const keyDivs = Array.from(section.querySelectorAll('.section-metadata > div > div:first-child'));
+  const styleDiv = keyDivs.find((keyDiv) => keyDiv.innerText.includes('style'));
+  const styleValues = styleDiv?.nextElementSibling?.textContent.toLowerCase();
+  const styles = styleValues?.split(', ').map((style) => style.replaceAll(' ', '-'));
+
+  return styles?.find((style) => style.includes('-up'));
+};
```

Comment:
I recently wrote a func to get metadata (aka 2 column table with key val pairs).  It's not checked in yet but you could add it to libs/utils.js:
```suggestion
const getMetadata = (el) => {
  if (!el) return {};

  const md = {};
  el.childNodes.forEach((n) => {
    const key = n.children?.[0]?.textContent?.toLowerCase();
    if (!key) return;

    const val = n.children?.[1]?.textContent?.toLowerCase();
    md[key] = val;
  });
  return md;
};

const getUpFromSectionMetadata = (section) => {
  const sectionMd = getMetadata(section.querySelector('.section-metadata'));
  const styles = sectionMd.style?.split(', ').map((style) => style.replaceAll(' ', '-'));
  return styles?.find((style) => style.includes('-up'));
};
```